submissions/ALT+F4 LTN-RLD-Cybersyn Mod Comparison/Requesting a Train.md

```diff
@@ -0,0 +1,115 @@
+
+One disadvantage to LTN is its learning curve. It can difficult to correctly configure stations as a new player, and it's rather to make mistakes that propagate through your base. Another complication is that LTN uses its own train limit signal, and uses that over vanilla train limits for its requester and its provider stations. However, that signal does not work with its depot stops, which instead require the use of vanilla train limits. (Optera, the mod author, has also stated that there no plans to change that)
+
+### Project Cybersyn - LTN Lite
+
+Named for [Project Cybersyn](https://en.wikipedia.org/wiki/Project_Cybersyn) of Allende's Chile, this mod released just a week after Rail Logistics Dispatcher. Project Cybersyn serves as something of a halfway point between LTN and RLD, removing some of LTN's more features in favor of simplicity.
+
+{% include image.html src='Cybersyn-combinator.png' alt='Project Cybersyn's Cybernetic Combinator next to a Train Stop' caption='Project Cybersyn's Cybernetic Combinator placed next to a Train Stop, highlighted to show the 1-tile range' %}
+
+Unlike LTN, which adds a whole new stop that must completely replace its vanilla counterpart, Project Cybersyn adds a new entity that must be placed within a tile of a station to control it. Together with a constant combinator to configure the new entity, the minimum footprint for the stop is 2x2 + 2x1 (or 1x2) + 1x1. Only the stop and Cybernetic Combinator are required for Depots and fuel stations. Sharp eyed readers will notice the 'No Power' icon - Cybersyn is only one of the three which adds an entity that requires power to function properly. Placing down more Cybernetic combinators allow one to use additional functions. One welcome change is that Cybersyn uses Vanilla Train Limits, unlike LTN.
+
+{% include image.html src='Cybersyn-GUI-1.png' alt='Cybernetic Combinator showing different modes' caption='Cybernetic Combinator showing different modes' %}
+
+Project Cybersyn's Cybernetic Combinators have five modes: Station, Depot, Refueler, Station Control, and Wagon Control. As with LTN, trains must first be sent to a depot in order for the mod to assign them a schedule. However, trains do not need to return to the depot in order to receive new schedule, thanks to the depot bypass feature. This feature may necessitate some changes in rail and depot design to take full advatage of.
+
+A second major change is that by default, trains with cargo in them are no longer allowed to receive new deliveries. LTN does warn the user if a train left with cargo, which could cause contamination in a base's logistics if the user did not take care to use filter inserters to unload cargo, or at best causing deliveries to grind to a halt as trains that already had inventory could not take on a full delivery. Cybersyn simply prevents that from happening in the first place. 
```

Comment:
it's a drastic change to the formatting, but you could try using subheaders here for each difference, e.g.

### Delivery differences

Trains with cargo..

### Depot bypass

Cybersyn's Refueler mode, combined with its depot bypass feature, ...